CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+# 1.2.0 (2020-04-9)
+## AWARENESS
```

Comment:
We should repeat some of our RC4 message to continue to raise awareness of the changes for anyone that's only paying attention to the full releases.